ehrapy/anndata_ext.py

```diff
@@ -192,6 +196,177 @@ def get_column_values(adata: AnnData, indices: int | list[int]) -> np.ndarray:
     return np.take(adata.X, indices, axis=1)
 
 
+def type_overview(
+    data: MuData | AnnData, sort_by: str | None = None, sort_reversed: bool = False
+) -> None:  # pragma: no cover
+    """Prints the current state of an :class:`~anndata.AnnData` or :class:`~mudata.MuData` object in a tree format.
+    Output could be printed in sorted format by using one of `dtype`, `order`, `num_cats` or `None`, which sorts by data type, lexicographical order,
+    number of unique values (excluding NaN's) and unsorted respectively. Note that sorting by `num_cats` only affects
+    encoded variables currently and will display unencoded vars unsorted.
+
+    Args:
+        data: :class:`~anndata.AnnData` or :class:`~mudata.MuData` object to display
+        sort_by: How the tree output should be sorted. One of `dtype`, `order`, `num_cats` or `None` (defaults to None -> unsorted)
+        sort_reversed: Whether to sort in reversed order or not
+
+    Example:
+        .. code-block:: python
+
+            import ehrapy as ep
+
+            adata = ep.dt.mimic_2(encode=True)
+            ep.anndata_ext.type_overview(adata)
+    """
+    if isinstance(data, AnnData):
+        _adata_type_overview(data, sort_by, sort_reversed)
+    elif isinstance(data, MuData):
+        _mudata_type_overview(data, sort_by, sort_reversed)
+    else:
+        print(f"[b red]Unable to present object of type {type(data)}. Can only display AnnData or MuData objects!")
```

Comment:
I totally get that the rich print looks nicer, but I feel like we should use actual errors whereever possible. They are used in the stacktrace and we're tuning them to be more readable with rich's stacktraceprint anyways.

Using a rich print here actually makes it harder to find the cause if the issue because it does not include the line where it occured etc.

Reply:
True, guess its a remainder from the old days ;)